Lib/test/support/__init__.py

```diff
@@ -2049,7 +2049,9 @@ def _run_suite(suite):
 
 # By default, don't filter tests
 _match_test_func = None
-_match_test_patterns = None
+
+_accept_test_patterns = None
+_ignore_test_patterns = None
```

Comment:
nitpick: i would prefer to use "ignore" everywhere, or "reject" everywhere, but not both.